### PR TITLE
Week 1-4 Block colours and activity descriptors

### PR DESCRIPTION
**Pull request by NathanDuran**
First 4 week to address issues #11 #17 #22

### CODE REVIEW

File: Learning_Materials/week_1/workbook1.ipynb
Comment (jim-smith) [inline Learning_Materials/week_1/workbook1.ipynb:1]: looks good 
- but a few typos and we should decide whether to use I or We
- you've merged 'how to get started' into the activity specifications. Does that mis making them long?
- should we split up the marks more within the activities ?

File: Learning_Materials/week_2/workbook2.ipynb
Comment (jim-smith) [inline Learning_Materials/week_2/workbook2.ipynb:1]: a fe typos and the breadth first search activity refers to depth first search

File: Learning_Materials/week_3/workbook3-heuristicSearch.ipynb
Comment (jim-smith) [inline Learning_Materials/week_3/workbook3-heuristicSearch.ipynb:1]: formatting is fine.
Could do with some markdown cells talking about how we have broken the stuff that tests down into a couple of functions 
- to introduce the idea  of creating workflows that let us rapidly compare algorithms

**Review by jim-smith — APPROVED** [review]
Some typos and things that need changing but formations is fine

**Comment by NathanDuran** [comment]
I don't think I can merge branches, possibly a permission problem? This branch has no conflicts now.

**Pull request merged**